mix.exs

```diff
@@ -42,6 +42,14 @@ defmodule Appsignal.Phoenix.MixProject do
 
   # Run "mix help deps" to learn about dependencies.
   defp deps do
+    system_version = System.version()
+
+    mime_dependency =
+      case Version.compare(system_version, "1.10.0") do
+        :lt -> [{:mime, "~> 1.0"}]
```

Comment:
```suggestion
        :lt -> [{:mime, "~> 1.0", only: [:test, :test_no_nif]}]
```

Mime is a transient dependency through plug, not a direct dependency. It's a dependency for which we only need to specify a specific version on older Elixir versions in the test environment.

If we were to publish the original change, the `appsignal_phoenix` package would have an additional dependency for mime dependending on what version of Elixir it was published from.